### PR TITLE
Create an OSL CentOS base image and switch everything to it / Ganeti Image

### PR DESCRIPTION
**Pull request by ramereth**
This finally provides a unified single image we can point all other images towards. This ensures the following happens:

- The repos are pointed to our mirrors for speed and reliability
- Install a few extra packages that seem to be common in most images
- Switch all current images to the new image
- Add a Ganeti image which can be useful in testing environments such as ganeti_webmgr

### CODE REVIEW

File: ganeti/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+ARG CENTOS_VER="systemd"
+FROM osuosl/centos:${CENTOS_VER}
```
Comment (doublej472) [inline ganeti/Dockerfile:2]: `systemd` centos version?

**Review by doublej472 — COMMENTED** [review]
Why does CentOS 7 get put into the `centos` directory directly, but CentOS 6 gets its own sub-directory? Is it some docker specific thing? It seems like an odd layout to use

File: ganeti/Dockerfile (hunk 1 above)
Comment (ramereth) [reply ganeti/Dockerfile:2]: This uses the [centos/systemd](https://hub.docker.com/r/centos/systemd/) container so you can run a container using systemd.

**Comment by ramereth** [comment]
@doublej472 by default the primary dockerfile will be CentOS 7. Since we're phasing out 6, I wanted to just create a sub directory for that version. It's not a docker specific thing but it's been kind of a standard I've seen other folks do.

**Review by doublej472 — APPROVED** [review]
LGTM!

**Pull request merged**